versioned_docs/version-v1.4/tutorials/access-application.md

```diff
@@ -1,6 +1,3 @@
----
-title: Access the application
----
```

Comment:
why deleted the header

Reply:
let me check it , perhaps it removed by reverting the mainline doc.